OptimizelySDKCore/OptimizelySDKCore/Optimizely.m

```diff
@@ -422,6 +431,9 @@ - (nullable NSString *)variableString:(nonnull NSString *)variableKey
         } else {
             [self.errorHandler handleError:variableUnknownForVariableKey];
         }
+        if (callback) {
+            callback(*error);
+        }
         
```

Comment:
The function signature starting line 412 passes both
```
                                error:(NSError * _Nullable * _Nullable)error
                             callback:(void (^)(NSError *))callback
```
OTOH, the function signature starting line 110 passes only
```
                    callback:(void (^)(NSError *))callback
```
Isn't this inconsistent style?

Why would the function signature starting line 412 want to return the "error"
if you can pass a "callback" in to handle the "error"?